README.md

```diff
@@ -151,15 +160,16 @@ When searching for service providers, the expected results is:
 
 #### Opportunities
 
-- Enhance testing capabilities and reusability.
+- Enhance testing capabilities and re-usability.
 - Implement a Database Provider.
 - Implement caching strategy.
 - Can you apply a better design pattern for the overall registration and data modeling?
-- Show us your understanding of Domain Driven Design and Loose Coupling.
+- Show us your understanding of Domain Driven Design and Loose Coupling?
```

Comment:
If you want to make it a question, then really make it a question. :thinking: 
This just sounds like you're kindly asking them to please do so if really they don't mind. :smile: